src/iframes/space-panel/space-panel.tsx

```diff
@@ -55,14 +55,14 @@ export const SpacePanel = () => {
           {roomsResult.spaceName}
         </PanelHeader>
       )}
-      {streamedHostsData && streamedHostsData.streamedHosts.length > 0 && (
+      {roomsResult && roomsResult.results.length > 0 && (
```

Comment:
may be we can create some constants out of render fn like `isRoomResultsExists` Also this could be done like `roomResult?.results?.length`

Reply:
i would like to have `roomResult?.results?.length` but unfortunately typings don't work well in this case :(
so here if i change to different boolean, or use optional chaining, type narrowing works bad and 
```
<ReplicatedNodes streamedHostsData={streamedHostsData} />
```
returns error. I know i can do `streamedHostsData as StreamedHostsData` but IMO this is more error-prone (manual type override)